stripe/test/test_resources.py

```diff
@@ -208,6 +208,60 @@ def test_retrieve(self):
         self.assertResponse(res)
 
 
+class ListObjectTests(StripeApiTestCase):
```

Comment:
We'll need to add tests for edge cases. Specifically, we need to cover the case when `has_more` is `True`. We also should test the case when we fetch an additional page and no results are returned.